### DISCUSSION TITLE
[Cache] ExpressionLanguage caching problem when using within Validators

### DISCUSSION THREAD

**Discussion by polina-suzdal** [polina-suzdal/symfony-discusion#5] [Q&A] [answered]
Symfony version(s) affected
6.2.2

Description
Since I've upgraded to Symfony 6.2.2 I'm experiencing caching issues when using custom Expressions withing Validators components in production environment, in particular when using:

Symfony\Component\Validator\Constraints\When in a DTO validation
Symfony\Component\Security\Http\Attribute\IsGranted for route access control

**Comment by powernic** [comment, marked as answer]
This also impacts me on 6.1.9. 

Because it has also been merged back into 6.1.9, 6.0.17 and 5.4.17 it's going to impact quite a lot of people 🙁
